docs/site/angular-auth-oidc-client/docs/documentation/configuration.md

```diff
@@ -219,6 +219,51 @@ export const appConfig: ApplicationConfig = {
 bootstrapApplication(AppComponent, appConfig);
 ```
 
+Additionally, you can use the feature function `withAppInitializerAuthCheck`
+to handle OAuth callbacks during app initialization phase. This replaces the
+need to manually call `OidcSecurityService.checkAuth(...)` or
+`OidcSecurityService.checkAuthMultiple(...)`.
+
+```ts
+import { ApplicationConfig } from '@angular/core';
+import { provideAuth, withAppInitializerAuthCheck } from 'angular-auth-oidc-client';
+export const appConfig: ApplicationConfig = {
+  providers: [
+    provideAuth(
+      {
+        config: {
+          /* Your config here */
+        },
+      },
+      withAppInitializerAuthCheck()
+    ),
+  ],
+};
+```
+
+If you prefer to manually check OAuth callback state, you can omit
+`withAppInitializerAuthCheck`. However, you then need to call
+`OidcSecurityService.checkAuth(...)` or
+`OidcSecurityService.checkAuthMultiple(...)` manually in your
+`app.component.ts` (or a similar code path that is called early in your app).
+
+```ts
+// Shortened for brevity
+...
+export class AppComponent implements OnInit {
+  private readonly oidcSecurityService = inject(OidcSecurityService);
+
+  ngOnInit(): void {
+    this.oidcSecurityService
+      .checkAuth()
+      .subscribe(({ isAuthenticated, accessToken }) => {
+        console.log('app authenticated', isAuthenticated);
+        console.log(`Current access token is '${accessToken}'`);
+      });
+  }
+...
+```
```

Comment:
We can also refer to the `checkAuth` API, this keeps the documentation in one place.

Reply:
Could you explain this in more detail? I'm not sure I fully understand what you mean.

Reply:
I can do that, but @FabianGosebrink requested to `add docs to have the "old" and "new" way side by side or underneath, so that we can link to it`.
I'm fine with either. Should I adapt, as suggested by @timdeschryver?

Reply:
Oh nevermind, my comment then :)

Reply:
Easy, I just wanted it to make it as easy as possible for the users to use. I am fine with both :)